src/data/equiv/algebra.lean

```diff
@@ -328,3 +327,75 @@ instance symm.is_ring_hom {e : α ≃r β} : is_ring_hom e.to_equiv.symm := hom
   e.symm.to_equiv x = e.to_equiv.symm x := rfl
 
 end ring_equiv
+
+-- We first define the type of automorphisms on groups, additive monoids,
+-- additive groups and rings using `mul_equiv`, `add_equiv` and `ring_equiv`.
+-- In each case this type is also a group.
+
+namespace group_aut
```

Comment:
For consistency, this should just be `group`.

Reply:
I thought you wanted me to take it out of ```group```?

Reply:
done